### PR TITLE
[Migrate] Fix missing created_at and wrong prerelease

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

**Comment by indirect** [comment]
@homu  r+

File: script/integration/migrate.rb [hunk 1]
```diff
@@ -85,6 +85,44 @@ def migrate_info_checksums(database_url)
   end
 end
 
+def migrate_created_at(database_url)
```
Comment (fotanus) [inline script/integration/migrate.rb]: There is actually an update on created_at at line 61. Quite hidden actually. Do you think it is better to remove that update?

Comment (segiddins) [inline script/integration/migrate.rb]: That's the whole point of this method

Comment (fotanus) [inline script/integration/migrate.rb]: But it runs after `migrate_checksums`,which populate some `create_at` and in this method you select versions with nil `created_at`. So if `migrate_checksums` is inserting wrong data, I think this won't fix? Or the data from `migrate_checksums` is legit but don't have for all versions?

**Pull request merged**